{{cookiecutter.github_project_name}}/setupbase.py

```diff
@@ -214,7 +273,10 @@ def recursive_mtime(path, newest=True):
     if os.path.isfile(path):
         return mtime(path)
     current_extreme = None
-    for dirname, _, filenames in os.walk(path, topdown=False):
+    for dirname, dirnames, filenames in os.walk(path, topdown=False):
+        # Don't recurse into node_modules
+        if 'node_modules' in dirnames:
+            dirnames.remove('node_modules')
```

Comment:
This will prevent detection of a stale build when a dependency has been updated. Is this intended?